modulus/utils/filesystem.py

```diff
@@ -43,11 +46,88 @@ def _get_fs(path):
         return fsspec.filesystem("file")
 
 
-def _download_cached(path: str, recursive: bool = False) -> str:
+def _download_ngc_model_file(path: str, out_path: str, timeout: int = 300) -> str:
```

Comment:
This is great! Now that we have a few ways to handle packages, I think we should better document them. Can we add something to the docs of the `Package` class to specify what all data sources are supported?

Reply:
Added doc string to package.